ansible/roles/test/files/tools/loganalyzer/loganalyzer_common_ignore.txt

```diff
@@ -262,6 +262,8 @@ r, ".* ERR syncd\d*#syncd:.*la_acl_key_profile_base::initialize failed to place
 r, ".* ERR syncd#syncd:.*SAI_LOG|SAI_API_TUNNEL: VLAN to VNI not implemented yet.*"
 r, ".* ERR syncd\d*#syncd: :- setEndTime: event 'create:SAI_OBJECT_TYPE_SWITCH:oid:0x[0-9a-fA-F]*' took \d* ms to execute.*"
 r, ".* ERR syncd.*#syncd: :- threadFunction: time span WD exceeded \d* ms for create:SAI_OBJECT_TYPE_SWITCH:oid:0x21000000000000.*"
+r, ".* ERR syncd.*#syncd.*logEventData:.*SAI_SWITCH_ATTR.*",
```

Comment:
I still feel add this in the common ignore list is not a good idea.
Could you check case test_container_autorestart.py, and do the similar thing here?

Reply:
Hi @kevinskwang, as discussed earlier, I don't think it makes sense to ignore these specifically for the drop counters TC, since these logs are related to config reload, and has nothing to do with that testcase specifically. Since we discussed that we will be modifying log event data error logs, to print as warnings in the future image (which is an image level change), modifying common loganalyzer will have the same consequence. We have already chosen to ignore timespan WD exceeded, and therefore the messages, which just contextualize that log, are not necessary, and will be confusing when captured in a sonic-mgmt run.